packager/blacklist.js

```diff
@@ -8,11 +8,14 @@
  */
 'use strict';
 
+var path = require("path"); // used for path separator
+
 // Don't forget to everything listed here to `testConfig.json`
 // modulePathIgnorePatterns.
 var sharedBlacklist = [
   __dirname,
   'website',
+  '/.git',             // added because nodeWatcher does not ignore hidden files
```

Comment:
Great, also noticed you fixed the NodeWatcher creation to pass an object with `glob` as a field, which seem to fix an issue where the glob pattern was being ignored.